### PR TITLE
Add scene transition from lobby to game

### PR DESCRIPTION
**Pull request by zhengHugo**

### CODE REVIEW

File: Assets/Scripts/Networking/MatchMessages.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using UnityEngine;
 using Mirror;
+using UnityEngine.SocialPlatforms.Impl;
```
Comment (eli-lam) [inline Assets/Scripts/Networking/MatchMessages.cs:5]: What is this for?

**Comment by zhengHugo** [comment]
To build, load the Lobby and Main scene additively. For client build, unload the main scene before build.

File: Assets/Scripts/Networking/MatchMessages.cs (hunk 1 above)
Comment (zhengHugo) [reply Assets/Scripts/Networking/MatchMessages.cs:5]: Not used. I'll remove this import.

**Review by eli-lam — APPROVED** [review]
Good job!

**Review by danielwiktorczyk — APPROVED** [review]
Went through this WIP quickly with @eli-lam . I didn't see anything game-breaking, and since it doesn't affect local code, approved!

**Pull request merged**